README.md

```diff
@@ -1,2 +1,100 @@
 # cake-git-ext-plugin
-Additional Git functionality like change detection for Cake scripts
+
+[![GitHub license](https://img.shields.io/github/license/ChrisTTian667/cake-git-ext-plugin.svg)](https://github.com/ChrisTTian667/cake-git-ext-plugin/main/LICENSE)
```

Comment:
Add a LICENSE file to this PR and link to it.

Reply:
There already is a Licence file linked below. But fixed the Link :)